vehicle-detection/darkflow/net/ops/convolution.py

```diff
@@ -85,8 +85,9 @@ def batchnorm(self, layer, inp):
                 'updates_collections' : None,
                 'is_training': layer.h['is_training']
                 })
-            v = tf.__version__.split('.')[1]
-            if int(v) < 12: key = 'initializers'
+            v_ma = tf.__version__.split('.')[0]
+            v_mi = tf.__version__.split('.')[1]
+            if int(v_ma) < 1 & int(v_mi) < 12: key = 'initializers'
```

Comment:
int(v_ma) < 1 & int(v_mi) < 12 , how can at a time value be less then 1 and 12, then second condition has no meaning.

is the actual condition like int(v_ma) > 1 & int(v_mi) < 12